srcpkgs/qgis/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'qgis'
+pkgname=qgis
+version=3.34.0
+revision=1
+archs="x86_64* i686"
```

Comment:
does this make more sense as nocross?

Reply:
I guess, I listed these because that's what other distros support for QGIS. Will give it a try.